src/main/java/core/basesyntax/service/handler/BalanceOperation.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service.handler;
+
+public class BalanceOperation implements OperationHandler {
+    @Override
+    public Integer handle(int quantity) {
+        return quantity;
+    }
```

Comment:
This solution will not scale well. You will not be able to support more complex logic. You should update the storage state in this method